app/client/ui2018/cssVars.ts

```diff
@@ -436,16 +436,19 @@ export const theme = {
     colors.darkGrey),
 
   /* Right Panel */
-  rightPanelTabFg: new CustomProp('theme-right-panel-tab-fg', undefined, colors.dark),
-  rightPanelTabBg: new CustomProp('theme-right-panel-tab-bg', undefined, colors.lightGrey),
+  rightPanelTabFg: new CustomProp('theme-right-panel-tab-fg', undefined, colors.slate),
+  rightPanelTabBg: new CustomProp('theme-right-panel-tab-bg', undefined, 'white'),
```

Comment:
Can you use colors.light defined above in the same file instead of white ?
It's easier to overload for self hoster whom use `static/custom.css`

```
  light: new CustomProp('color-light', '#FFFFFF'),
```

Reply:
oops my bad, saw a lot of hard-coded `white` already there so I assumed there were none defined, missed it! thanks